File: backends/python-fastapi/app/domain/client/condition.py
"""
クライアントドメイン 検索条件モジュール。

Author: Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
"""
from dataclasses import dataclass
from typing import Optional


@dataclass
class ClientCondition:
    """クライアント検索条件。"""
    keyword: Optional[str] = None
    status: Optional[int] = None

File: backends/ts-hono/src/domain/client/condition.ts
/**
 * クライアントドメイン 検索条件モジュール。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */

/** クライアント一覧の検索条件。 */
export interface ClientCondition {
  keyword?: string;
  status?: number;
}
